rendering/markup_content_test.go

```diff
@@ -0,0 +1,33 @@
+package rendering
+
+import "testing"
+import "github.com/stretchr/testify/assert"
+import "github.com/stretchr/testify/require"
+
+func TestGetDefaultMarkupFromNil(t *testing.T) {
```

Comment:
Okay, agreed. they just seemed so small, tiny, and cozy. Those poor little tests looked like they should be sleeping under a shared blanket. :)